frontend/src/scenes/events/volume-definitions/VolumeTable.tsx

```diff
@@ -32,6 +31,44 @@ const search = (sources: VolumeTableRecord[], searchQuery: string): VolumeTableR
         .search(searchQuery)
         .map((result) => result.item)
 }
+export function Owner({ ownerId, user }: { ownerId?: number | null; user?: UserBasicType }): JSX.Element {
```

Comment:
If possible, would be great if you could use `TeamMemberID` from https://github.com/PostHog/posthog/pull/4574/files

Reply:
Looks like there are a lot of similar components between these two PRs

Reply:
Also, the UI 

> Looking good 🥳!! Made some code suggestions, nothing big. UI/UX comments below. Please feel free to push back.
> 
> 1. You could do this in a separate PR but we need to change the tabs titles to better reflect what's going on.
> 2. I think we can drop the stats columns if stats are not enabled and then just keep the notice at the top (adjusting copy and reducing prominence).
> 3. When clicking to add description I would just make use of keyboard shortcuts (esc & cmd + enter) and maybe just icons instead of the full buttons. This will also prevent the jumping around of the page when adding descriptions.
> 4. I should only be able to edit the description of an event at a time (otherwise it leads to some weird behavior when submitting).
> 5. I should not be able to add a description to PostHog events. Further, the description of these events should already be loaded and the popover (when hovering) not shown. This will help with the mental model. I also think it's not worth opening a drawer for these events, wdyt?
> 6. The drawer should open when I click anywhere on the row (except to edit the description).
> 7. The last modified / last modified by attributes do not show if they're null but this makes the layout awkward. We should either handle a `null` case (e.g. "Unknown") or center the rest of the props.
> 8. The UI seems a bit off, think it's the margins & font sizes (see for example `CohortsDetailRow.tsx` in https://github.com/PostHog/posthog/pull/4574/files), happy to propose some changes in a separate PR (let me know).
> 9. Tag autocomplete is not working. Doesn't suggest already existing tags.
> 10. Think it's worth adding the tags column to the table too, will help with organization a ton.
> 11. Worth adding blank space at the bottom of the drawer so you can scroll well all the way down.
> 12. Finally (and could be for a separate PR), think it's worth adding some simple E2E tests for the basic behavior here.

Two questions:

1. What tabs are we referring to here? The panels with `Usage`, `General` ?
8. I agree that margins and font sizes are slightly off, I tried to use reuseable styling for it but I wasn't sure which to choose from so I ended up using `<Title>`, what do you recommend here? There seems to be a couple of different ways we choose to standardize our styling in the app. I assume here we want to stick with antd's <Row> and <Col> and use the `mt, md` classNames?

Reply:
It looks like that PR is still a draft right now so I'll merge what I have in first and will refactor it once that PR is in, since I'll probably be still finishing up some things for this drawer?

Reply:
Perfect, sounds good! You may just want to actually recreate the component from that PR here as I'm not sure how far we're from being able to merge the other PR.